bundles/binding/org.openhab.binding.isy/src/main/java/org/openhab/binding/isy/ISYActiveBinding.java

```diff
@@ -0,0 +1,718 @@
+		for (ISYBindingProvider provider : this.providers) {
+			ISYBindingConfig config = provider
+					.getBindingConfigFromItemName(itemName);
+
+			if (config != null) {
+				return config;
+			}
+		}
+
+		return null;
+	}
+
+	/**
+	 * Translate the openHAB command into a Insteon command
+	 * 
+	 * @param address
+	 *            Insteon address of device/node/group/scene
+	 * @param command
+	 *            openHAB command
+	 */
+	private void processCommand(final ISYBindingConfig config,
+			final Command command) {
+		if (!this.insteonClient.isISYReady()) {
+			this.logger.warn("ISY is not ready");
+
+			return;
+		}
+
+		UDNode node = null;
+
+		// find out whether node is group or node
+		try {
+			node = this.insteonClient.getNodes().get(config.getController());
+		} catch (Exception e) {
+			this.logger.warn("no such device, " + config.getController(), e);
+		}
+
+		if (node == null) {
+			try {
+				node = this.insteonClient.getGroups().get(
+						config.getController());
+			} catch (Exception e) {
+				this.logger.warn(
+						"no such group found, " + config.getController(), e);
+			}
+		}
+
+		if (node != null) {
+			executeCommand(config, node, command);
+		} else {
+			this.logger.warn("No node found for address "
+					+ config.getController());
+		}
+	}
+
+	/***
+	 * Performs the execution of the command in the ISY unit. It will handle the
+	 * translation of the OpenHab command to the ISY version.
+	 * 
+	 * @param config
+	 *            Configuration item for the item in OpenHab
+	 * @param node
+	 *            The node in ISY that represents the binding configuration.
+	 * @param command
+	 *            The OpenHab command to execute.
+	 */
+	private void executeCommand(final ISYBindingConfig config,
+			final UDNode node, final Command command) {
+
+		// Do percent first so dimmers dim and do not just go on and off. 
+		if(command instanceof IncreaseDecreaseType && ISYNodeType.DIMMER.equals(config.getType())) {
+			IncreaseDecreaseType type = (IncreaseDecreaseType) command;
+			
+			switch (type) {
+			case INCREASE:
+				if (node instanceof UDGroup) {
+					this.insteonClient.brightenScene(node.address);
+				} else {
+					this.insteonClient.brightenDevice(node.address);
+				}
+				break;
+			case DECREASE:
+				if (node instanceof UDGroup) {
+					this.insteonClient.dimScene(node.address);
+				} else {
+					this.insteonClient.dimDevice(node.address);
+				}
+				break;
+			}
+		} else if (command instanceof OnOffType) {
+			OnOffType type = (OnOffType) command;
+
+			switch (type) {
+			case ON:
+				if (ISYNodeType.THERMOSTAT.equals(config.getType())) {
+					// turn thermostat Fan State to On
+					this.insteonClient.changeNodeState(ISYControl.CLIFS.name(),
+							"7", node.address);
+
+					// provide immediate feedback such that the UI gets updated
+					// or else the switch button will not be refreshed until the
+					// new state has been reached
+					this.eventPublisher.postUpdate(config.getItemName(), type);
```

Comment:
This is the default behavior in openHAB, that commands sent to items automatically update the item's state, unless the binding code or the user explicitly disables autoupdate processing.  By explicitly posting the update here, you are overriding the user's ability to specify `autoupdate="false"` to control whether the switch is updated, so please remove.